linux-anvil-ppc64le/Dockerfile

```diff
@@ -17,14 +17,14 @@ RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-AltArch-7-ppc64le
 
 # Install basic requirements.
+COPY scripts/yum_clean_all /opt/docker/bin/
 RUN yum update -y && \
     yum install -y \
         bzip2 \
         sudo \
         tar \
         which && \
-    yum clean all && \
-    rm -rf /var/cache/yum/*
+    /opt/docker/bin/yum_clean_all
```

Comment:
Maybe we should just move all of this into that script.

Reply:
Yes, this and other parts can/should be consolidated. But let's do further cleanup.maintenance work later, not in this PR.